docs/_locale/de/LC_MESSAGES/settings.po

```diff
@@ -106,6 +106,30 @@ msgstr "Unter *Zeitzone* konfigurieren Sie die Anzeige von Zeiten in Signavio Wo
 msgid "*Email signature* replaces the default *Workflow Accelerator team* signature at the bottom of :ref:`notification emails <notifications>`."
 msgstr "Unter *Email Signatur* ersetzen Sie die *Signavio Workflow Team*-Standardsignatur, die mit den :ref:`Signavio Workflow Accelerator-Emailbenachrichtigungen<notifications>` verschickt wird."
 
+msgid "Process creation"
+msgstr "Prozesserstellung"
+
+msgid "*Process creation* You can activate this option to configure which group of users can create processes."
```

Comment:
Ok, then we should update the screenshots and call the group "Process creators" or something